### DISCUSSION TITLE
Fresh install: some missing translations

### DISCUSSION THREAD

**Discussion by beatwiz** [studio1902/statamic-peak#204] [Q&A] [answered]
First of all: this is awesome. Thank you so much.

But then, this happened. 

I double checked and the translations are there in en/lang.php. 

Any ideas what I may be missing? Everything is set-up normally as per instructions.

![Xnip2022-03-18_13-55-51](https://user-images.githubusercontent.com/8360167/159015959-3d495d90-9a9f-4e00-ba7e-6fede47ee210.jpg)

**Comment by robdekort** [comment, marked as answer]
Hey thanks for the kind words! Happy to hear.

When you're using Laravel 9 the strings should be in `/lang/`. For Laravel 8 they need to be in `/resources/lang/`. Does that help?

**Reply by beatwiz** [reply]
Insta-solution! 
It seems I haven't fiddled enough with L9 yet. :)
Thanks so much @robdekort 
Have a great weekend!

**Reply by robdekort** [reply]
Nice. Happy it helped!
